system/Cookie/Cookie.php

```diff
@@ -660,12 +662,12 @@ public function offsetGet($offset)
 
 		if ($offset === 'httponly')
 		{
-			return $this->httpOnly;
+			return $this->httponly;
 		}
 
 		if ($offset === 'samesite')
 		{
-			return $this->sameSite;
+			return $this->samesite;
 		}
```

Comment:
Since you made all property names as lowercase, then there's no  need for these 2 guards here. Also in offsetExists method, no need to check for the two. Only check for `expire`.